etcdctl/ctlv3/command/get_command.go

```diff
@@ -19,7 +19,9 @@ import (
 	"strings"
```

Comment:
I'm not especially enthusiastic about supporting this soft-failure mode as a one-off feature that only fetches all keys since it's considerable baggage and doesn't fit well with the rest of the system. For example, should delete range also have a soft-fail mode like this? Txns? Watches?

If it's worth having auth soft-fail, I feel it should be a client wrapper so that it can be used anywhere. For instance, etcdctl could take a flag `--auth-soft-fail` that would replace the client with the soft fail client so every command supports it.